source/filtered_table_model.ts

```diff
@@ -35,15 +39,15 @@ export class FilteredTableModel extends TableModel {
    *  already being processed, then the sub-transaction gets consolidated into
    *  the parent transaction.
    */
-  public beginTransaction(): void {
+  private beginTransaction(): void {
```

Comment:
Private methods are not documented, are placed after all public methods, and generally omit the return type.